qutebrowser/mainwindow/mainwindow.py

```diff
@@ -327,16 +327,30 @@ def resizeEvent(self, e):
     def closeEvent(self, e):
         """Override closeEvent to display a confirmation if needed."""
         confirm_quit = config.get('ui', 'confirm-quit')
-        count = self._tabbed_browser.count()
-        if confirm_quit == 'never':
+        tab_count = self._tabbed_browser.count()
+        download_manager = objreg.get('download-manager', scope='window',
+                                      window=self.win_id)
+        download_count = download_manager.rowCount()
+        quit_texts = []
+        # Close if set to never ask for confirmation (backward compatible)
+        if confirm_quit == 'never' or 'never' in confirm_quit:
```

Comment:
The `confirm_quit == 'never'` and `(backward compatible)` are obsolete now that it's always a list.